.github/workflows/cargo_bloat.yml

```diff
@@ -0,0 +1,83 @@
+name: Binary size report
+on:
+  push:
+    branches: [ develop ]
+    paths:
+      - 'libraries/**/*.rs'
+      - 'src/**/*.rs'
+      - 'patches/**'
+      - '**/Cargo.toml'
+      - '.cargo/config'
+      - '!third_party/**'
+  pull_request:
+    branches: [ develop ]
+    types: [opened, synchronize, reopened]
+
+jobs:
+  cargo_bloat:
+    runs-on: ubuntu-18.04
+    steps:
+      # First run: PR
+      - uses: actions/checkout@v2
+        with:
+          submodules: "true"
+      - name: Install Rust toolchain
+        run: rustup show
+      - uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
+      - name: Install Python dependencies
+        run: python -m pip install --upgrade pip setuptools wheel
+      - name: Set up OpenSK
+        run: ./setup.sh
+      - uses: actions-rs/cargo@v1
+        with:
+          command: install
+          args: cargo-bloat
+      - name: Run bloat on the PR
+        run: .github/workflows/bloat_output.sh "Output for cargo bloat including the PR" "bloat_output.txt"
+
+      - name: Copy script
+        run: cp .github/workflows/bloat_output.sh ${{ runner.temp }}/bloat_output.sh
+      - name: Copy text
+        run: cp bloat_output.txt ${{ runner.temp }}/bloat_output.txt
+
+      # Second run: PR
+      - uses: actions/checkout@v2
+        with:
+          submodules: "true"
+          ref: "develop"
```

Comment:
I think this could be `${{ github.base_ref }}` if we want to support the workflow for stable too.

However, it doesn't work for `on.push`, it only works for `on.pull_request`. I'm not sure but I guess `${{ github.base_ref || github.ref_name }}` might work for both `on.push` and `on.pull_request`.

Reply:
Since stable is never actually doing anything new, I was wondering if we want this information on stable. But maybe good to confirm that nothing changed, compared to expectation from develop.